scheduler/src/main/java/io/camunda/zeebe/scheduler/ActorThread.java

```diff
@@ -160,6 +160,11 @@ public static ActorThread ensureCalledFromActorThread(final String methodName) {
     return thread;
   }
 
+  public static boolean isCalledFromActorThread() {
```

Comment:
:wrench: Similar methods exists: `ActorControl#isCalledFromWithinActor` and `ActorControl#ensureCalledFromActorThread`. I think it makes sense to keep these methods together in one class.